python/cugraph/cugraph/structure/property_graph.py

```diff
@@ -1760,7 +1766,11 @@ def edge_props_to_graph(
             # Ensure a valid edge_weight_property can be used for applying
             # weights to the subgraph, and if a default_edge_weight was
             # specified, apply it to all NAs in the weight column.
-            if edge_weight_property in edge_prop_df.columns:
+            if edge_weight_property == self.type_col_name:
```

Comment:
Can you update the comment above, since the code it's describing is different now? Same for the MG version.

Reply:
done